test/propose.lean

```diff
@@ -73,8 +73,6 @@ info: Try this: have : p ≠ 1 := ne_one hp
 ---
 info: Try this: have : p ∣ p * p ↔ p ∣ p ∨ p ∣ p := dvd_mul hp
 ---
-info: Try this: have : p ∣ a := Prime.dvd_of_dvd_pow hp h
----
```

Comment:
I don't quite understand why this change to `induction'` weakens this test for `have?!` slightly. At least the dropped suggestion is not one of the two important ones.

@semorrison should this be looked at before merging?

Reply:
Ah nvm, this is fine. After starting induction, the hypothesis is now `h : p ∣ 1` and one would need to turn it into `h : p ∣ a ^ 0` before being able to use `Prime.dvd_of_dvd_pow`. And that's not something `have?!` is supposed to figure out by itself, for one because `a` is ambiguous in that step.